app/controllers/create_survey.rb

```diff
@@ -0,0 +1,53 @@
+get '/surveys/new' do
+  erb :survey_new
+end
+
+
+post '/surveys' do
```

Comment:
If it's the case that you have to be logged in to get to this route, then add a guard clause at the top to either redirect the user to somewhere else (e.g. login), or throw a relevant error/status code (e.g. 403 Forbidden), or take them to a specific view or error page saying why they can't do what they're trying to do.

One simple example:

```ruby
redirect '/login' unless logged_in?
```